.vale/styles/RedHat/Headings.yml

```diff
@@ -148,80 +152,87 @@ exceptions:
   - Netty
   - Newdoc
   - Nginx
+  - Node.js
   - NuGet
   - OAuth
+  - ORM
   - OmniSharp
+  - OpEx
   - OpenID Connect
   - OpenJDK
   - OpenShift
   - OpenTracing
-  - OpEx
-  - ORM
   - PHP
   - Podman
   - Podman Desktop
-  - PostgreSQL
   - PostScript
+  - PostgreSQL
   - PowerPC
   - PowerShell
   - Productize
   - Productized
-  - Proof Key for Code Exchange
   - Prometheus
+  - Proof Key for Code Exchange
   - Pytorch
-  - qeth
-  - Quarkus
+  - Qt
   - Quarkiverse
+  - Quarkus
   - Quiltflower
   - Qute
+  - RESTEasy
   - Red Hat
-  - Redis
   - RedBoot
+  - Redis
   - Redistributions
-  - RESTEasy
   - Rolfe
-  - Sakila
   - SCM
+  - SDK
   - SELinux
+  - SVG
+  - Sakila
   - Semeru
-  - SmallRye
   - Shadowman
+  - SmallRye
+  - Software Development Kit
+  - Software Development Kits
   - Spotify
   - StarOffice
-  - startx
   - Suchow
-  - SVG
-  - System
   - SysV
+  - System
   - Tekton
   - Telekom
   - Tensorflow
   - Texinfo
   - Toolset
   - Traefik
-  - Uber
-  - UltraSPARC
   - URI
   - URIs
   - URL
   - URLs
+  - Uber
+  - UltraSPARC
   - Vale
   - Valgrind
   - Velero
   - Vert.x
-  - vNIC
-  - vNUMA
   - WebAuthn
+  - WebSocket
   - Webview
   - Webviews
-  - WebSocket
   - Wildfly
   - Window Maker
   - Woopra
   - Wordpress
   - XEmacs
-  - xterm
   - Yana
   - Yeoman
   - Zowe
-  - GitLab Code Quality
+  - cgroup
```

Comment:
Are lines 231 to 238 needed? I suggest removing and single-sourcing a text file of all camel/pascal case items, similar to how GitLab docs do Vale rules using links to single-sourced text files.

Reply:
Yes because they could start a sentence and must be lower case.